tests/conftest.py

```diff
@@ -240,6 +240,30 @@ def structure_folder(test_folder):
     return test_folder / "calculations" / "structures"
 
 
+@pytest.fixture
+def workflow_structure_folder(test_folder):
+    """
+    Fixture to provide the path to the example file.
+
+    Returns
+    -------
+        Path: The path to the example file.
+    """
+    return test_folder / "workflows" / "structures"
```

Comment:
Do these want to be fixtures or just module level constants?

Possibly want them to be `fixture(scope="session")` to avoid wasting recomputation (incredibly minor, but still)

Reply:
They probably could be module level constants, but it's also convenient having these collected. Is there much overhead to a fixture? If not (with the scope added) I think I'd leave it.